galvan-support/pom.xml

```diff
@@ -46,12 +47,14 @@ limitations under the License.
       <groupId>org.terracotta.internal</groupId>
       <artifactId>client-runtime</artifactId>
       <version>${project.version}</version>
+      <scope>provided</scope>
     </dependency>
     <dependency>
       <groupId>org.terracotta.internal</groupId>
       <artifactId>terracotta-kit</artifactId>
       <type>zip</type>
       <version>${project.version}</version>
+      <scope>test</scope>
```

Comment:
Created #575 where I realised this dependency can be fully removed so I suggest either to update this PR or to merge the other one.